keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1057,13 +1116,8 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: 
-- [ ] API .status
-  - Condition name: 
-  - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+- [x] Events
+  - Event Reason: Pod running event
```

Comment:
elaborate on the "doesn't work" case here too please.  It will come up for every upgraded cluster during the transition.

Reply:
Updated this section to outline that we re-use the existing events for image pull. This will not change with the graduation.